lib/backend/repositories/user_repo.dart

```diff
@@ -151,6 +155,13 @@ class UserRepo {
   Future<UserProfile> updateUser(
       Map<String, dynamic> user, String userAddress) async {
     final result = await _userService.updateUser(user, userAddress);
+    final box = await Hive.openBox(HiveBoxes.kAppBox);
+    box.put(HiveKeys.kUserData, jsonEncode(result));
+    final Map<String, dynamic> decodedUserData =
+        jsonDecode(await box.get(HiveKeys.kUserData));
+    decodedUserData['user'] = result;
+    box.delete(HiveKeys.kUserData);
+    box.put(HiveKeys.kUserData, jsonEncode(decodedUserData));
```

Comment:
Why putting this data twice?

Reply:
We’re not, we’re updating a key on the object.